packages/kbn-eslint-plugin-eslint/index.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
   rules: {
     'require-license-header': require('./rules/require_license_header'),
     'disallow-license-headers': require('./rules/disallow_license_headers'),
+    'no-default-export': require('./rules/no_default_export'),
```

Comment:
overall +1 from me for the PR, as today I spent some time trying to add another custom rule while porting https://github.com/elastic/kibana/pull/33562

just curious - why we use custom `no_default_export` rule instead of https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md ? 
our doesn't handle `export { foo as default };` for example

Reply:
Probably because it didn't exist at the time, but I'm not sure. I'd love to review a PR that moves to the rule from eslint-plugin-import!

Reply:
ok 👍